package-res/resources/web/prompting/components/StaticAutocompleteBoxComponent.js

```diff
@@ -173,6 +180,10 @@ define([ 'common-ui/util/util', 'cdf/components/BaseComponent',  'amd!cdf/lib/jq
       } else {
         return val;
       }
+    },
+
+    _createLabelTag: function(source) {
+      return $("<a></a>").html(source);
```

Comment:
'href' used in the code that was moved from Report Viewer: https://github.com/pentaho/pentaho-platform-plugin-reporting/pull/295/commits/f26b0f74db574bb450ccb0ae7539d3a1e11fad8a
In additionally if I understand correctly jQuery UI autocomplete uses li with href by default.